src/main/java/org/apache/sling/feature/cpconverter/accesscontrol/SystemUser.java

```diff
@@ -25,10 +25,12 @@
     private final String id;
 
     private final RepoPath path;
+    private final RepoPath intermediatePath;
 
-    public SystemUser(String id, RepoPath path) {
+    public SystemUser(String id, RepoPath path, RepoPath intermediatePath) {
```

Comment:
alternatively, use the old constructor and call `path.getParent()` in the constructor.... i don't recall if RepoPath.getParent can return null but that might be worth verify if one wants to be on the safe side.

Reply:
Right, I think it is slightly nicer to have it be done outside the constructor - just in case one wants to use a different path vs. intermediatePath set-up where there is no direct parent relationship. However, I totally agree it is overkill for the issue at hand and maybe it would be better todo it inside the constructor. Let's keep it like this for now.